assets/training/finetune_acft_hf_nlp/environments/acpt/context/requirements.txt

```diff
@@ -14,7 +14,7 @@ rouge-score==0.1.2
 sacrebleu==2.2.1
 bitsandbytes==0.41.0
 einops==0.6.1
-cryptography==41.0.2
+cryptography==41.0.3
```

Comment:
_mltable_ already installs cryptography. Can we check which version gets installed from mltable before explicitly installing the package?

Reply:
Verified that the latest mltable will get cryptography 41.0.3. So, removing it

Log after pip installing the latest mltable:

Successfully installed cryptography-41.0.3 mltable-1.4.1Successfully installed cryptography-41.0.3 mltable-1.4.1